components/form-builder/app/navigation/ShareDropdown.tsx

```diff
@@ -56,24 +55,24 @@ export const ShareDropdown = () => {
             className={`${menuWidth} rounded-lg px-1.5 py-1 shadow-md bg-white border-1 border-black`}
           >
             {!name && (
-              <DropdownMenuPrimitive.Item>
-                <span className="flex-grow ml-2 text-sm inline-block w-[400px]">
-                  {t("share.missingName.message1")}{" "}
-                  {/* note: using a Button here ... using a Link doesn't close the menu */}
-                  <Button
-                    theme="link"
-                    className="inline-block"
-                    onClick={() => {
-                      // Timeout is currently required because menu steals focus on close.
-                      setTimeout(() => {
-                        getRef("fileNameInput")?.current?.focus();
-                      }, 50);
-                    }}
-                  >
-                    {t("share.missingName.message2")}
-                  </Button>{" "}
-                  {t("share.missingName.message3")}
+              <DropdownMenuPrimitive.Item
```

Comment:
This seemed the simplest fix.  @timarney if there is a better way, just let me know. The sub men needs to be "activate-able" by enter. Some way to let the user know the menu is "clickable" and what will happen when s/he clicks it would be helpful but not necessarily required